<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageTags</class>
 <widget class="QWidget" name="ImageTags">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QListView" name="listImages">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DropOnly</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="imageType"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Description</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="imageDescription"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QToolButton" name="buttonExport">
       <property name="toolTip">
        <string>Export selected images to disk</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
       <property name="icon">
        <iconset theme="document-save">
         <normaloff>../../../.designer/backup</normaloff>../../../.designer/backup</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonAdd">
       <property name="toolTip">
        <string>Import images from disk</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>../../../.designer/backup</normaloff>../../../.designer/backup</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonDelete">
       <property name="toolTip">
        <string>Remove selected images</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>../../../.designer/backup</normaloff>../../../.designer/backup</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>122</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="cloneImages">
       <property name="toolTip">
        <string>Apply image set to all selected files</string>
       </property>
       <property name="text">
        <string>Apply to all</string>
       </property>
       <property name="icon">
        <iconset theme="edit-copy">
         <normaloff>../../../.designer/backup</normaloff>../../../.designer/backup</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listImages</tabstop>
  <tabstop>imageType</tabstop>
  <tabstop>imageDescription</tabstop>
  <tabstop>buttonExport</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>buttonDelete</tabstop>
  <tabstop>cloneImages</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
